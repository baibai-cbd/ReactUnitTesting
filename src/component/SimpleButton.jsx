import React from 'react';

class SimpleButton extends React.Component {



    render() {
        return (
          <div>
            <button disabled={true}>Click me</button>
          </div>
        );
    }
}

export default SimpleButton;