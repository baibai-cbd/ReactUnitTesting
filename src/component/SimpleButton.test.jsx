import React from 'react';
import SimpleButton from './SimpleButton';
import { shallow } from 'enzyme';


describe('SimpleButton', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<SimpleButton />);
    expect(wrapper).toMatchSnapshot();
  });
});