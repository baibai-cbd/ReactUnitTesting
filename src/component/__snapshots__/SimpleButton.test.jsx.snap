// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleButton renders correctly 1`] = `
<div>
  <button
    disabled={true}
  >
    Click me
  </button>
</div>
`;
